dojo/api_v2/permissions.py

```diff
@@ -1,17 +1,42 @@
 import re
-
 from rest_framework.exceptions import ParseError
+from dojo.importers.reimporter.utils import get_target_engagement_if_exists, get_target_product_if_exists, get_target_test_if_exists
 from dojo.models import Endpoint, Engagement, Finding, Product_Type, Product, Test, Dojo_Group
 from django.shortcuts import get_object_or_404
-from rest_framework import permissions
+from rest_framework import permissions, serializers
 from dojo.authorization.authorization import user_has_permission
 from dojo.authorization.roles_permissions import Permissions
 
 
+def get_import_meta_data_from_dict(data):
```

Comment:
This method is not related to permissions but is used in the serializers and the views as well. It should be moved to a `dojo/api_v2/utils.py` module.

Reply:
it was in importer/utils.py but because it generates serializer.ValidationError I didn't want to have it too deep in there. I think I'll move it to the serializers module